dashboard/components/FragmentGraph.tsx

```diff
@@ -99,7 +99,7 @@ export default function FragmentGraph({
   planNodeDependencies: Map<string, d3.HierarchyNode<PlanNodeDatum>>
   fragmentDependency: FragmentBox[]
   selectedFragmentId?: string
-  channelStats?: Map<string, ChannelStatsDerived>
+  channelStats?: Map<string, ChannelDeltaStats>
```

Comment:
Prometheus can directly return `ChannelStatsDerived`, since we can directly do rate calculation within the `promql`. So we construct a new proto definition for this format and call it `ChannelDeltaStats`. For embedded metrics, after all the calculation, it also produces the semantically equivalent `ChannelDeltaStats`. So we can just unify both of them.